src/sagemaker/jumpstart/types.py

```diff
@@ -736,6 +736,334 @@ def _get_regional_property(
         return alias_value
 
 
+class JumpStartBenchmarkStat(JumpStartDataHolderType):
+    """Data class JumpStart benchmark stats."""
+
+    __slots__ = ["benchmark_name", "instance_type", "latency_ms"]
+
+    def __init__(self, spec: Dict[str, Any]):
+        """Initializes a JumpStartBenchmarkStat object
+
+        Args:
+            spec (Dict[str, Any]): Dictionary representation of benchmark stat.
+        """
+        self.from_json(spec)
+
+    def from_json(self, json_obj: Dict[str, Any]) -> None:
+        """Sets fields in object based on json.
+
+        Args:
+            json_obj (Dict[str, Any]): Dictionary representation of benchmark stats.
+        """
+        self.benchmark_name: str = json_obj["benchmark_name"]
+        self.instance_type: str = json_obj["instance_type"]
+        self.latency_ms: int = json_obj["latency_ms"]
```

Comment:
IIUC, this could be an ever-expanding set of benchmark stats (i.e. latency, cpu utilization, gpu utilization, etc.), we should confirm the expected rate of change for Quicksilver to ensure enumerating every field is going to be a scalable approach.